src/SelectInput/SelectInput.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+// @ts-ignore
+import { storiesOf } from '@storybook/react'
+import { withState } from '../utils/tests/decorators/StateDecorator'
+import { Dropdown } from '../Dropdown'
+
+import { SelectInput } from './'
+import styled, { css } from 'styled-components'
+
+const Row = styled.div<{ columns?: number | false }>`
+  display: grid;
+  grid-gap: 20px;
+  ${props => props.columns !== false && css`
+    grid-template-columns: repeat(${props => props.columns || 10}, fit-content(200px));
+  `}
+`
+
+export default storiesOf('forms|SelectInput', module)
+  .addDecorator(withState({
+    isOpen: false
+  }))
+  .add('default', ({ state, storeState }) => (
+    <Row columns={1}>
+      <Dropdown useButtonWidth open={state.isOpen} button={
+        <SelectInput placeholder='Choose option' onClick={() => storeState({ ...state, isOpen: !state.isOpen })}/>
```

Comment:
I was half expecting this component to take care of creating the `Dropdown` component and handling the open state internally, wouldn't that be easier when using this component?

```
<SelectInput placeholder={...} data={[...]} value={...} onChange={value => void} />
```

Reply:
no - this was just a quick example - I made a comment on top of this comment that we need to make a select component and then we can combine selectInput and dropdown and a select to make a single component. Not part of this pr.

Reply:
I read that. Was under the impression that the SelectInput component would be that wrapper.